types/truncate-utf8-bytes/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for truncate-utf8-bytes 1.0
+// Project: https://github.com/parshap/truncate-utf8-bytes#readme
+// Definitions by: Ben Limmer <https://github.com/blimmer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace truncateUtf8Bytes;
```

Comment:
The package does not say it can be used in html as is. So this shouldnt be present
```suggestion
```